src/datalad_installer.py

```diff
@@ -984,11 +984,41 @@ class NeurodebianComponent(Component):
         ],
     )
 
+    KEY_FINGERPRINT = "0xA5D32F012649A5A9"
+    DOWNLOAD_SERVER = "us-nh"
+
     def provide(self, extra_args: Optional[List[str]] = None, **kwargs: Any) -> None:
         log.info("Installing & configuring NeuroDebian")
         log.info("Extra args: %s", extra_args)
         if kwargs:
             log.warning("Ignoring extra component arguments: %r", kwargs)
+        release = readcmd("lsb_release", "-cs").strip()
```

Comment:
shouldn't we do all this extra work only if `neurodebian` package is not available? 
also it is dependent but not entirely orthogonal -- there might be no `neurodebian` package available (missed debian/ubuntu release and we forgot to provide backport) but have neurodebian enabled which could be judged by

```
root@93218c4c7afe:/# apt-cache policy | grep o=NeuroDebian
     release o=NeuroDebian,a=focal,n=focal,l=NeuroDebian,c=main,b=amd64
     release o=NeuroDebian,a=data,n=data,l=NeuroDebian,c=main,b=amd64
```
so, it is only if `neurodebian` package not available and no NeuroDebian apt repository is configured already - configure it manually (before trying to do `apt install neurodebian` which could still fail).